sdk/contentsafety/Azure.AI.ContentSafety/src/Customization/TextBlocklist.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using Azure.Core;
+
+namespace Azure.AI.ContentSafety
+{
+    /// <summary> Text Blocklist. </summary>
+    public partial class TextBlocklist
+    {
+        private string _description;
+        private bool _isDiscriptionChanged = false;
+
+        /// <summary> Initializes a new instance of TextBlocklist. </summary>
+        /// <param name="blocklistName"> Text blocklist name. </param>
+        /// <param name="description"> Text blocklist description. </param>
+        internal TextBlocklist(string blocklistName, string description)
+        {
+            BlocklistName = blocklistName;
+            _description = description;
```

Comment:
In the generated code, it is `Description = description;` I customized it to `_description = description;` because I don't want to set `_isDiscriptionChanged`. But there is a problem. Consider the roundtrip scenario: we get a `TextBlocklist` from service with payload `{description: null}`, and then send it to service without changing, the payload would be `{}`.  Let me know if it is OK.

Reply:
@pshao25, why would the service returning a value for description in the response cause us to send it in the `Update` payload if the end-user never changes the value?

Reply:
It's rare, and maybe won't happen. I'm just considering all the corner cases.

Reply:
I think that violates the PATCH contract, though.  Sending `null` for a property tells the service to delete the property.  Is that the desired behavior if the caller does nothing?